shim/opentelemetry-opencensus-shim/test-requirements.txt

```diff
@@ -0,0 +1,36 @@
+asgiref==3.7.2
+attrs==23.2.0
+cachetools==5.3.3
+certifi==2024.2.2
+charset-normalizer==3.3.2
+Deprecated==1.2.14
+flaky==3.7.0
+google-api-core==2.17.1
+google-auth==2.28.1
+googleapis-common-protos==1.62.0
+grpcio==1.62.0
+idna==3.6
+importlib-metadata==6.11.0
+iniconfig==2.0.0
+opencensus==0.11.1
+opencensus-context==0.1.3
+opencensus-proto==0.1.0
+packaging==23.2
+pluggy==1.4.0
+protobuf==3.20.3
+py==1.11.0
+py-cpuinfo==9.0.0
+pyasn1==0.5.1
+pyasn1-modules==0.3.0
+pytest==7.1.3
+pytest-benchmark==4.0.0
+requests==2.31.0
+rsa==4.9
+six==1.16.0
+tomli==2.0.1
+typing_extensions==4.10.0
+urllib3==2.2.1
+wrapt==1.16.0
+zipp==3.17.0
```

Comment:
Shouldn't we add `-e opentelemetry-api` as well?

Reply:
Right, fixed.